plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -2799,6 +2799,10 @@ ul.wc_coupon_list_block {
 		border: 0 none;
 		float: none;
 	}
+
+	.select2-container {
```

Comment:
On smaller screen sizes, the input width overflows the container, making the "value" input hidden on mobiel.

<img width="481" alt="Screenshot 2025-02-18 at 1 17 19 PM" src="https://github.com/user-attachments/assets/c3e9c427-5454-4a24-ba1a-2db1f4e8f04c" />
<img width="549" alt="Screenshot 2025-02-18 at 1 17 06 PM" src="https://github.com/user-attachments/assets/e9763e34-6fd4-4998-8004-0247c2098a39" />

Reply:
@joshuatf, I'm having trouble reproducing this.  Can you confirm what browser/screen size this was happening with?

Reply:
The above screenshot is from Brave and affects most screen sizes under 1000px.  Things are better on Chrome, though there are some issues with margin consistency and the value field being a bit too small on mobile screen sizes:

<img width="400" alt="Screenshot 2025-02-26 at 5 14 03 PM" src="https://github.com/user-attachments/assets/261c697f-3f8d-48b6-b89d-07068ddebd86" />
<img width="481" alt="Screenshot 2025-02-26 at 4 58 37 PM" src="https://github.com/user-attachments/assets/2a63dbd9-e59f-40b1-98ed-6578c65dfdfc" />

This is what it looks like on `trunk`:
<img width="395" alt="Screenshot 2025-02-26 at 5 14 35 PM" src="https://github.com/user-attachments/assets/53bdd271-e611-4919-bb74-d2088faa4415" />